compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Repl.hs

```diff
@@ -743,7 +743,7 @@ buildModule file imports decls = L dummySrcSpan HsModule
   , hsmodExports = Nothing
   , hsmodImports = Preprocessor.onImports $
       noLoc (simpleImportDecl (mkModuleName "Daml.Script")) :
-      noLoc (simpleImportDecl (mkModuleName "GHC.Types")) :
+      noLoc (simpleImportDecl (mkModuleName "GHC.Tuple.Check")) :
```

Comment:
Please see https://github.com/digital-asset/ghc/pull/144 for suggestion to avoid needing to import `GHC.Tuple.Check`